packages/react/src/components/LoaderOverlay/index.tsx

```diff
@@ -9,34 +9,35 @@ interface LoaderOverlayProps extends React.HTMLAttributes<HTMLDivElement> {
   label?: string;
 }
 
-const LoaderOverlay = React.forwardRef<
-  HTMLDivElement,
-  React.HTMLAttributes<HTMLDivElement>
->(({ className, variant, label, ...other }: LoaderOverlayProps, ref) => (
-  <div
-    className={classNames(
-      'Loader__overlay',
-      className,
-      variant === 'large'
-        ? 'Loader__overlay--large'
-        : variant === 'small'
-        ? 'Loader__overlay--small'
-        : ''
-    )}
-    ref={ref}
-    {...other}
-  >
-    <div className="Loader__overlay__loader">
-      <Loader variant={variant} />
-      <AxeLoader />
+const LoaderOverlay = React.forwardRef<HTMLDivElement, LoaderOverlayProps>(
+  ({ className, variant, label, ...other }: LoaderOverlayProps, ref) => (
+    <div
+      className={classNames(
+        'Loader__overlay',
+        className,
+        variant === 'large'
+          ? 'Loader__overlay--large'
+          : variant === 'small'
+          ? 'Loader__overlay--small'
+          : ''
+      )}
+      ref={ref}
+      {...other}
+    >
+      <div className="Loader__overlay__loader">
+        <Loader variant={variant} />
+        <AxeLoader />
+      </div>
+      {label ? <span className="Loader__overlay__label">{label}</span> : null}
+      {other.children}
     </div>
-    {label ? <span className="Loader__overlay__label">{label}</span> : null}
-    {other.children}
-  </div>
-));
+  )
+);
```

Comment:
All that changed here was `React.HTMLAttributes<HTMLDivElement>` -> `LoaderOverlayProps`